ddpui/core/orguserfunctions.py

```diff
@@ -74,8 +74,8 @@ def lookup_user(email: str):
 
 def signup_orguser(payload: OrgUserCreate):
     """create an orguser and send an email"""
-
     signupcode = payload.signupcode
+    print("Valid codes:", os.getenv("SIGNUPCODE"), os.getenv("DEMO_SIGNUPCODE"))
```

Comment:
we dont use print statements. we have common logger, check other files you will get an idea.

can you please replace it then ? thank you. great job on this